### ISSUE TITLE
schema: The $id and title are not generated using the Vulkan Header API version

### ISSUE THREAD

**Issue by christophe-lunarg** [KhronosGroup/Vulkan-Profiles#135]
The generated schema file has the following content:
```
    "$id": "https://schema.khronos.org/vulkan/profiles-1.3.203.json#",
    "title": "Vulkan Profiles Schema for Vulkan 1.3.203",
```
But it's not correct because the current Vulkan Header is 1.3.204, hence we should have:
```
```
    "$id": "https://schema.khronos.org/vulkan/profiles-1.3.204.json#",
    "title": "Vulkan Profiles Schema for Vulkan 1.3.204",
```
So it would be great to generate these lines using the Vulkan Header API version instead of these being hard coded.

**Comment by aqnuep** [comment]
Not sure how I missed this, but should be an easy fix.

**Comment by aqnuep** [comment]
Fix is in PR #137.
